packages/fxa-auth-server/grunttasks/ftl.js

```diff
@@ -7,7 +7,7 @@
 module.exports = function (grunt) {
   grunt.config('concat', {
     ftl: {
-      src: ['lib/senders/emails/auth.ftl', 'lib/**/senders/emails/**/en.ftl'],
+      src: ['lib/l10n/auth.ftl', 'lib/**/senders/emails/**/en.ftl'],
```

Comment:
I moved `auth.ftl` to live inside `lib/l10n` instead because that felt like a better spot for it since it's no longer just for emails. I also added the strings from `session.js` to live in `auth.ftl` rather than a new `en.ftl` file here.